code/modules/mob/living/carbon/human/species/working_joe/_species.dm

```diff
@@ -1,14 +1,3 @@
-/datum/species/synthetic/colonial/working_joe
```

Comment:
Why are you moving this

Reply:
I moved it back to the location it was originally in.

Reply:
#3792 intentionally moved all WJ code into its own directory, move the code back into that directory